scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/simulation/IncrementalSimulationDriver.java

```diff
@@ -149,7 +150,7 @@ public SimulationResults getSimulationResultsUpTo(Duration endTime){
     }
 
     if(lastSimResults == null || endTime.longerThan(lastSimResultsEnd)) {
-      lastSimResults = SimulationEngine.computeResults(
+      lastSimResults = SimulationDriver.computeResults(
```

Comment:
I'm debating whether or not it's a problem to have one driver depend on the other. I suspect we'll want to split out the results computation into its own class.